core/domain/stats_jobs_one_off.py

```diff
@@ -802,6 +802,20 @@ def entity_classes_to_map_over(cls):
     @classmethod
     def require_non_negative(
             cls, exp_id, exp_version, property_name, value, state_name=None):
+        """Yields the Negative count of the exploration according
+        to the passed property name.
+
+        Args:
+            exp_id: str. Exploration ID.
+            exp_version: str. Exploration version.
+            property_name: str. Property name.
+            value: dict. Values of different properties for the exploration.
+            state_name: str. State name of the exploration.
+
+        Yields:
+            str. Formatted String containing the information of the negative
```

Comment:
Why is "String" capitalized?

Also see my suggestion for the main docstring; you'll want to do something similar here. The phrase "containing the information of the negative count value" is constructed a bit clumsily, and developers probably won't understand what it means.